README.md

```diff
@@ -1 +1,61 @@
-# prometheus-exporter
\ No newline at end of file
+# Memgraph metrics exporter
+
+[Memgraph](https://github.com/memgraph/memgraph) exporter for Prometheus.
+
+The metrics currently collected are:
+
+- **TODO**
+
+## Running the exporter
+
+```shell
+$ git clone https://github.com/memgraph/prometheus-exporter.git
+$ cd prometheus-exporter
+$ python3 main.py
+```
+
+## Basic Prometheus Configuration
+
+Add Memgraph target to the Prometheus `scrape_configs`:
+
+```yaml
+scrape_configs:
+  - job_name: 'memgraph-metrics'
+  static_configs:
+    - targets: ['<<MEMGRAPH_EXPORTER_HOST>>:8000']
+```
+Adjust the host accordingly.
+
+_A simple Prometheus configuration [prometheus.yml](config/prometheus.yml)._
+
+## Running and Debugging the Setup
+
+1. Startup your local Memgraph instance.
+2. Make sure Memgraph is sending the metric information
+```shell
+$ curl localhost:9092
+```
+You should see a JSON object containing the metrics information.
+
+3. Run the Memgraph exporter
+```shell
+$ cd prometheus-exporter
+$ python3 main.py
```

Comment:
I would also document that in case the user changed the default port for our HTTP server from 9092, or if user is running from Docker Memgraph, something might need to change. From Docker probably not, but I am not 100% sure, but if port is changed, then `main.py` will not work.

Reply:
Yeah, there is a comment [above](https://github.com/memgraph/prometheus-exporter/pull/2#issuecomment-1540148513) where I list some thing we could improve. It really depends on the scope of this feature.
As for what needs to change in case you run from Docker or change the port. Basically the address is hard-coded in the main.py, so based on your setup you would need to update it. Same goes for the Prometheus address.